File: src/main/java/net/stefankrause/ChromedriverTest.java
package net.stefankrause;

import org.openqa.selenium.*;
import org.openqa.selenium.chrome.*;

public class ChromedriverTest {
    public static void main(String[] args) throws Exception {
        System.setProperty("webdriver.chrome.driver", "node_modules/chromedriver/lib/chromedriver/chromedriver");

        WebDriver driver = new ChromeDriver();
        driver.get("http://www.google.com/ncr");
        WebElement searchBox = driver.findElement(By.id("lst-ib"));
        searchBox.sendKeys("ChromeDriver");
        searchBox.submit();
        Thread.sleep(5000);
        driver.quit();
    }
}

File: chromedrivertest.js
var webdriver = require('selenium-webdriver'),
    By = webdriver.By,
    until = webdriver.until;

var driver = new webdriver.Builder()
    .forBrowser('chrome')
    .setChromeOptions()
    .build();

driver.get("http://www.google.com/ncr");
driver.findElement(By.id("lst-ib")).then(elem => {
    elem.sendKeys("ChromeDriver");
    elem.submit();
});
driver.wait(until.titleIs('ChromeDriver - Google Search'), 1000);
driver.quit();
